Expecto.Tests/FocusedTests.fs

```diff
@@ -0,0 +1,77 @@
+module Expecto.Focused
```

Comment:
Should have equal sign after module name? Or no indentation of the text below (preferable)